book/views.py

```diff
@@ -23,3 +25,44 @@ def get (self, request):
             publication_date__range=[previous_days,today])
             .order_by('-publication_date')[:int(limit)])]
         return JsonResponse({"oneMonthBook":books}, status=200)
+
+class BookDetailView(View):
+    def get(self, request, book_id):
+        try :
+            book = Book.objects.select_related('category').prefetch_related('review_set').get(id=book_id)
+            book_detail = {
+                'title'            : book.title,
+                'subtitle'         : book.subtitle,
+                'image_url'        : book.image_url,
+                'company'          : book.company,
+                'author'           : book.author,
+                'contents'         : book.contents,
+                'company_review'   : book.company_review,
+                'page'             : book.page,
+                'publication_date' : book.publication_date,
+                'description'      : book.description,
+                'category'         : book.category.name,
+                'review_count'     : book.review_set.count(),
+                'reder'            : book.userbook_set.count()
+                }
+            return JsonResponse({'book_detail':book_detail}, status =200)
+        except Book.DoesNotExist:
+            return JsonResponse({'message':'NOT_EXIST_BOOK'}, status =400)
```

Comment:
```suggestion
            return JsonResponse({'book_detail':book_detail}, status=200)
        except Book.DoesNotExist:
            return JsonResponse({'message':'NOT_EXIST_BOOK'}, status=400)
```
파일 내부에 전반적으로 `status_code` 띄어쓰기가 통일되어 있지 않네요 :) 모두 같은 스타일로 통일하시는 것을 추천드립니다